src/Core/Table/ArcTable.fs

```diff
@@ -372,42 +391,49 @@ type ArcTable(name: string, headers: ResizeArray<CompositeHeader>, values: Syste
 
     // - Column API - //
     // GetColumnAt?
-    member this.GetColumn(columnIndex:int) : CompositeColumn =
+    member this.GetColumn(columnIndex:int, ?fillDefault : bool) : CompositeColumn =
```

Comment:
fillDefault behaves different to my expectations. Did you discuss this with someone else in the department and decided that we fail if a cell does not exist inside row/column count? I thought we only fail for trying to access outside these params?

Reply:
I mean, with the `fillDefault` flag every user can decide. I could add some comment to make it more clear. Or would you wish to reverse the default behavious, and have a `failIfMissing` flag instead?